test/inputoutput/test_io.py

```diff
@@ -1,3 +1,13 @@
-from main.inputoutput.io import inc
+from main.inputoutput.io import inc,unzip_file
+import os
+
 def test_answer():
     assert inc(3) == 5
+
+def test_unzip_file():
+    fromPath=r"test\data\Desktop.zip"
+    toPath=r"test\data\target"
+    curDir=os.getcwd()
+    fromPath=os.path.join(str(curDir),str(fromPath))
+    toPath=os.path.join(str(curDir),str(toPath))
+    assert unzip_file(fromPath,toPath) == 1
```

Comment:
Ideally this assert should check if the files exist in the output location and match what was in the zip file. 

For this, you can keep zipped and unzipped version of the files in data, extract the zipped file in 'target' and then compare it with the unzipped version.